modules/utility/init.zsh

```diff
@@ -8,23 +8,27 @@
 
 if (( terminfo[colors] >= 8 )); then
 
-  # ls Colours
-  if (( ${+commands[dircolors]} )); then
+  if (( ${+commands[dircolors]} )) && ls --version &>/dev/null; then
     # GNU
 
-    (( ! ${+LS_COLORS} )) && if [[ -s ${HOME}/.dir_colors ]]; then
+    # ls Colours
+    if [[ ${+LS_COLORS} -eq 0 && -s ${HOME}/.dir_colors ]]; then
       eval "$(dircolors --sh ${HOME}/.dir_colors)"
     else
       export LS_COLORS='di=1;34:ln=35:so=32:pi=33:ex=31:bd=1;36:cd=1;33:su=30;41:sg=30;46:tw=30;42:ow=30;43'
     fi
-
     alias ls='ls --group-directories-first --color=auto'
+    alias lx='ll -X' # long format, sort by extension
+
+    # Always wear a condom
+    alias chmod='chmod --preserve-root -v'
+    alias chown='chown --preserve-root -v'
```

Comment:
Hmm, this should actually not be here, because it's inside the `(( terminfo[colors] >= 8 ))` section...